assets/app/views/create.html

```diff
@@ -7,67 +7,117 @@
-          <form name="from_source_form" class="form-inline gutter-bottom">
-            <p>Create your application from a Git source code repository. Optionally specify a branch, tag, or commit ID with #ref.</p>
-            <div class="row">
-              <div class="col-md-8">
-                <div class="input-group">
-                  <span ng-class="{'has-warning': from_source_form.from_source_url.$dirty && !sourceURLPattern.test(from_source_url)}">
-                    <input class="form-control input-lg"
-                      name="from_source_url"
-                      placeholder="Repository URL"
-                      type="text"
-                      required
-                      aria-describedby="from_source_help"
-                      ng-model="from_source_url">
-                  </span>
-                  <span class="input-group-btn">
-                    <button type="submit" class="btn btn-lg btn-default" ng-click="createFromSource()"
-                      ng-disabled="from_source_form.$invalid" value="">Next</button>
-                  </span>
-                </div>
-                <div id="from_source_help" class="help-block">For example, https://github.com/openshift/nodejs-ex#master
-                  <a href="#" ng-click="from_source_url = 'https://github.com/openshift/nodejs-ex#master'"
-                    style="margin-left: 3px;">Try it out!<i class="fa fa-level-up" style="margin-left: 3px; font-size: 17px;"></i></a></div>
-                <span class="text-warning" ng-if="from_source_form.from_source_url.$dirty && !sourceURLPattern.test(from_source_url)">Source repository should be a URL</span>
-              </div>
-            </div>
-          </form>
+          <p ng-if="emptyCatalog && !loaded">Loading...</p>
+
+          <div ng-if="emptyCatalog && loaded && !nonBuilderImages.length" class="empty-state-message">
+            <h2 class="text-center">No images or templates.</h2>
+
+            <p class="gutter-top">
+              No images or templates are loaded for this project or the shared
+              <code>openshift</code> namespace. An image or template is
+              required to add content.
+            </p>
+
+            <p>
+              To load an image stream or template from a file, run the command
+              <div><code>oc create -f &lt;filename&gt; -n {{projectName}}</code></div>
+            </p>
+
+            <p><a href="{{projectName | projectOverviewURL}}">Back to overview</a></p>
+          </div>
+
+          <!-- Show an abbreviated message if only non-builders exist in the project.
+               We show a link to view the non-builder images below. -->
+          <p ng-if="emptyCatalog && loaded && nonBuilderImages.length">No builder images or templates.</p>
+
+          <div ng-show="!emptyCatalog">
+            <h1>Select Image or Template</h1>
+
+            <p>Choose from web frameworks, databases, and other components.
```

Comment:
Yeah, the "You'll be prompted..." was there before. I can take it out.

We use "Get started..." on the empty project page. I don't think we should repeat it again.